includes/admin/class-wc-rest-payments-disputes-controller.php

```diff
@@ -125,4 +153,26 @@ public function close_dispute( $request ) {
 		$dispute_id = $request->get_param( 'dispute_id' );
 		return $this->forward_request( 'close_dispute', [ $dispute_id ] );
 	}
+
+	/**
+	 * Extract disputes filters from request
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 */
+	private function get_disputes_filters( $request ) {
+		return array_filter(
+			[
+				'match'             => $request->get_param( 'match' ),
+				'store_currency_is' => $request->get_param( 'store_currency_is' ),
+				'created_before'    => $request->get_param( 'date_before' ),
+				'created_after'     => $request->get_param( 'date_after' ),
+				'created_between'   => $request->get_param( 'date_between' ),
+				'status_is'         => $request->get_param( 'status_is' ),
+				'status_is_not'     => $request->get_param( 'status_is_not' ),
```

Comment:
This mapping answer my[ previous comment ](https://github.com/Automattic/woocommerce-payments/pull/3609/files#r783910952) but I'm not sure if this is the best approach. 

In Transactions this mapping was used to support legacy parameter name. I think it would be much better to simply use actual filter names through out entire codebase - both php and js.

Reply:
Updated.